app/src/main/res/values/styles.xml

```diff
@@ -47,6 +47,12 @@
     <item name="listBackground">@drawable/list_bg</item>
     <item name="selectedBackground">@drawable/list_bg_light</item>
     <item name="android:itemBackground">@color/primary_light</item>
+    <item name="drawerBackground">@color/drawer_background_day</item>
+    <item name="dividerColor">@color/divider_light</item>
+    <item name="android:textColorPrimary">@color/primary</item>
+    <item name="android:textColorSecondary">@color/primary</item>
+    <item name="android:textColorTertiary">@color/primary</item>
```

Comment:
What are your opinions on using lighter shades for secondary and tertiary colors?

Reply:
@abdulwd So, I have changed the text color according to what is usually recommended. Please review it.